system.go

```diff
@@ -28,6 +28,32 @@ type SystemInput struct {
 	Note        *string          `json:"note,omitempty"`
 }
 
+func (s *System) GetTag(tagId ID, client *Client) *Tag {
```

Comment:
We should also reverse the order of these arguments - in other golang tools the client and/or context for the client is usually the first argument.